requirements-dev.txt

```diff
@@ -1,3 +1,18 @@
 cython
 libsass
 pre-commit
+jsonschema
+psutil
+python-ldap
+lxml
+websockify
+m2crypto
+cheetah3
+CherryPy
+python-pam
+pyOpenSSL
+requests
+mock
+pyyaml
+pep8
+pyflakes
```

Comment:
Usually pip has the latest version of python modules which may not be true for some Linux distributions. So adding all those dependencies to be installed from pip may lead in conflicts when running wok from a package.

I'd recommend to only use pip when it is really needed.